pkg/config/config.go

```diff
@@ -197,12 +200,36 @@ func (fc *FulcioConfig) GetVerifier(issuerURL string, opts ...InsecureOIDCConfig
 		}
 	}
 
-	// If this issuer hasn't been recently used, or we have special config options, then create a new verifier
-	// and add it to the LRU cache.
+	// clone rather than modifying the default transport
+	var transportClone *http.Transport
```

Comment:
Can we simplify this block of code? I'd conditionally initialize the client if `iss.CACert != ""` with the transport, and drop using Clone.